types/leaflet/index.d.ts

```diff
@@ -579,6 +580,32 @@ export class ImageOverlay extends Layer {
 
 export function imageOverlay(imageUrl: string, bounds: LatLngBoundsExpression, options?: ImageOverlayOptions): ImageOverlay;
 
+export interface VideoOverlayOptions extends ImageOverlayOptions {
+    autoplay?: boolean;
+    loop?: boolean;
+}
+
+export class VideoOverlay extends Layer {
```

Comment:
Why don't you use ImageOverlay for `extends` here?

Reply:
Link to the actual code: https://github.com/Leaflet/Leaflet/blob/master/src/layer/VideoOverlay.js#L24

Reply:
Because  `imageOverlay.getElement` returns `HTMLImageElement`, but `videoOverlay.getElement` returns `HTMLVideoElement`.
Maybe, it will be better to define `basedImageOverlay` for the common functions and `ImageOverlay` and `VideoOverlay` will extend it.

Reply:
No I don't think so... I'm quite happy, that we get rid of all the `base...` interfaces. Can you add a short comment describing the reason for this, please?

Reply:
Ok

Reply:
OK